crates/nu-std/README.md

```diff
@@ -3,36 +3,49 @@
+╭────┬─────────────────────────────┬────────────────────────────────────────────────────────────────╮
+│  # │            name             │                                usage                           │
+│  0 │ std assert                  │ Universal assert command                                       │
+│  1 │ std assert equal            │ Assert $left == $right                                         │
+           . . .
+│ 11 │ std clip                    │ put the end of a pipe into the system clipboard.               │
+│ 12 │ std dirs add                │ Add one or more directories to the list.                       │
+           . . .
+├────┼─────────────────────────────┼────────────────────────────────────────────────────────────────┤
+│  # │            name             │                                usage                           │
+╰────┴─────────────────────────────┴────────────────────────────────────────────────────────────────╯
 ```
 
-## :pencil2: contribute to the standard library
-- all the commands of the standard_library are located in [`std.nu`](std.nu)
-- the tests are located in files that have a name starting with "test_", e.g. [`test_std.nu`](test_std.nu)
-- a test runner, at [`tests.nu`](tests.nu), allows to run all the tests automatically
+## :toolbox: Using the standard library in the REPL or in scripts
+All commands in the standard library must be "imported" into the running environment 
+(the interactive read-execute-print-loop (REPL) or a `.nu` script) using the
+[`use`](https://nushell.sh/commands/docs/use.html) command.
 
-### :wrench: add new commands
-- add new standard commands by appending to [`std.nu`](std.nu)
-- add associated tests to [`test_std.nu`](tests_std.nu) or preferably to `test_<submodule>.nu`.
-    - define a new exported (!) `test_<feature>` command
-    - import the `assert` functions you need at the top of the functions, e.g. `use std.nu "assert eq"`
+You can choose to import the whole module, but then must refer to individual commands with a `std` prefix, e.g:
+```
+use std
+ . . .
+std log debug "Running now"
+std assert (1 == 2)
+```
 
-### :test_tube: run the tests
-the following call should return no errors
-```bash
-NU_LOG_LEVEL=DEBUG cargo run -- -c "use std; std run-tests --path crates/nu-std"
+Or you can enumerate the specific commands you want to import and invoke them without the `std` prefix.
+```
+use std ["log debug" assert]`
+. . .
+log debug "Running again"
+assert (2 == 1)
 ```
 
-> **Warning**  
-> the `cargo run --` part of this command is important to ensure the version of `nushell` and the version of the library are the same.
+This is probably the form of import you'll want to add to your `env.nu` for interactive use.
+
+## :pencil2: contribute to the standard library
+You're invited to contribute to the standard library! 
+See [CONTRIBUTING.md](https://github.com/nushell/nushell/blob/main/crates/nu-std/CONTRIBUTING.md) for details.
```

Comment:
```diff
-See [CONTRIBUTING.md](https://github.com/nushell/nushell/blob/main/crates/nu-std/CONTRIBUTING.md) for details.
+See [CONTRIBUTING.md](CONTRIBUTING.md) for details.
```
should work.

Reply:
Actually, no.  Your proposed change won't work in local repo, only online, thanks to github magic.  I think the risk that the online target will disappear in the future is low.

Reply:
oooh that is for local repos :open_mouth: :+1:

Reply:
Hmm, do we want the link to open, or do we want the path to be obvious?

Reply:
the absolute HTTP path is good :+1:

Reply:
:heavy_check_mark: